core/src/script/CGXP/plugins/FeaturesGrid.js

```diff
@@ -276,8 +276,12 @@ cgxp.plugins.FeaturesGrid = Ext.extend(cgxp.plugins.FeaturesResult, {
      */
     csvEncoding: 'UTF-8',
 
-    /** private: attribute[selectAll]
```

Comment:
I don't know why this line exists. I've thought that this is an old thing and have just remove that.